app/utils/config.js

```diff
@@ -3,11 +3,16 @@ const path = require('path');
 // require and configure dotenv, will load vars in .env in PROCESS.ENV
 const envPath = path.resolve(__dirname, '../.env');
 require('dotenv').config({ path: envPath });
-const isProduction = new RegExp('production').test(process.env.NODE_ENV);
-if (isProduction) {
-	process.env.SLACK_CALLBACK = 'https://saltlakedsa.org/auth/slack/callback';
-} else {
-	process.env.SLACK_CALLBACK = 'http://localhost:3111/auth/slack/callback';
+if (!process.env.SLACK_CALLBACK) {
+	const isProduction = new RegExp('production').test(process.env.NODE_ENV);
+	const isStaging = new RegExp('test').test(process.env.NODE_ENV);
+	if (isProduction) {
+		process.env.SLACK_CALLBACK = 'https://saltlakedsa.org/auth/slack/callback';
+	} else if (isStaging) {
+		process.env.SLACK_CALLBACK = 'http://18.237.248.6:3111/auth/slack/callback';	
```

Comment:
if this doesn't work in staging, we should restrict the allowed slack callback url list to 'https://saltlakedsa.org/auth/slack/callback' in the slack console, and only use slack auth in production.